### PR TITLE
add /_users to couchdb

### PR DESCRIPTION
**Pull request by josesolaro**

### CODE REVIEW

File: Audit/Controllers/AuditController.cs [hunk 1]
```diff
@@ -21,45 +21,79 @@ public class AuditController : ControllerBase
...
+            try
+            {
+                _logger.LogInformation(String.Format("Deleting: {0}", id));
+                var doc = await _repository.FindById(id, false).ConfigureAwait(true);
+                var result = await _repository.Delete(doc).ConfigureAwait(true);
+                return result;
+            }
+            catch (Exception e)
+            {
+                return false;
```
Comment (nestorreloaded) [inline Audit/Controllers/AuditController.cs:100]: maybe log the exception here?

File: Audit/Repository/Repository.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            {
+                _logger.LogInformation("Tables creation skip.");
+                return false;
+            }
+        }
+        public async Task<bool> IsUp()
+        {
+            try
+            {
+                if (await ConnectionDB.IsUpAsync())
```
Comment (nestorreloaded) [inline Audit/Repository/Repository.cs:90]: can't we use something like

return await ConnectionDB.IsUpAsync() to avoid the if sentence?

**Review by nestorreloaded — CHANGES_REQUESTED** [review]

**Review by nestorreloaded — APPROVED** [review]

File: Audit/Controllers/AuditController.cs [hunk 3]
```diff
@@ -21,45 +21,80 @@ public class AuditController : ControllerBase
...
         public async Task<JsonResult> Post([FromBody] Entity.Audit value)
         {
-            return new JsonResult(await _repository.Create(value).ConfigureAwait(true));
+            try
+            {
+                return new JsonResult(await _repository.Create(value).ConfigureAwait(true));
+            }
+            catch (Exception e)
+            {
+                return new JsonResult(e.Message);
```
Comment (exe135) [inline Audit/Controllers/AuditController.cs:67]: We can return a logger message here, as on Delete(string id).

**Review by exe135 — CHANGES_REQUESTED** [review]

File: Audit/Controllers/AuditController.cs (hunk 3 above)
Comment (josesolaro) [reply Audit/Controllers/AuditController.cs:67]: Done

**Review by exe135 — APPROVED** [review]

**Pull request merged**